files/zh-cn/web/api/extendablecookiechangeevent/changed/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: ExtendableCookieChangeEvent：changed 属性
+slug: Web/API/ExtendableCookieChangeEvent/changed
+l10n:
+  sourceCommit: 60c3843f55839380e0c0cdc293ea694fe9943158
+---
+
+{{securecontext_header}}{{APIRef("Cookie Store API")}}{{AvailableInWorkers("service")}}
+
+{{domxref("ExtendableCookieChangeEvent")}} 接口的 **`changed`** 只读属性返回给定 `ExtendableCookieChangeEvent` 实例已变更的 cookie。
+
+## 值
+
+一个包含已变更的 cookie 的对象数组。每个对象包含以下属性：
+
+- `name`
+  - : 记录 cookie 名称的字符串。
+- `value`
+  - : 记录 cookie 的值的字符串。
+- `domain`
+  - : 记录 cookie 域名的字符串。
+- `path`
+  - : 记录 cookie 路径的字符串。
+- `expires`
+  - : {{glossary("Unix time", "Unix 时间戳")}}（以毫秒为单位表示），记录 cookie 的到期日期。
+- `secure`
+  - : 一个布尔值，表示 cookie 是否仅在安全上下文中使用（HTTPS 还是 HTTP）。
```

Comment:
英文文档的描述并不准确（请帮忙更新一下英文文档）

```suggestion
  - : 一个{{jsxref("boolean", "布尔值", "", 1)}}，表示 cookie 是否仅在安全上下文中使用（HTTPS 还是 HTTP）。
```

Reply:
> 并不准确（请帮忙更新一下英文文档）

没太明白不准确指的是？

英文原文：

> "A {{jsxref("boolean")}} indicating whether the cookie is from a site with a secure context (HTTPS rather than HTTP).”

Reply:
```md
A {{jsxref("boolean", "", "", 1)}}
```
改成这样？

Reply:
> indicating whether the cookie is from a site with a secure context (HTTPS rather than HTTP)

该属性并不用于标识当前的 cookie 是否是来自具有安全上下文的站点的。它是用于标识当前 cookie 仅能在安全上下文中使用。

Reply:
烦请更新一下上游的文档